impl/src/test/java/org/ehcache/impl/internal/store/heap/OnHeapStoreKeyCopierTest.java

```diff
@@ -58,18 +58,8 @@
 
   private static final Key KEY = new Key("WHat?");
   public static final String VALUE = "TheAnswer";
-  public static final NullaryFunction<Boolean> NOT_REPLACE_EQUAL = new NullaryFunction<Boolean>() {
-    @Override
-    public Boolean apply() {
-      return false;
-    }
-  };
-  public static final NullaryFunction<Boolean> REPLACE_EQUAL = new NullaryFunction<Boolean>() {
-    @Override
-    public Boolean apply() {
-      return true;
-    }
-  };
+  public static final Supplier<Boolean> NOT_REPLACE_EQUAL = () -> false;
```

Comment:
Here it is. Can be used at many place in the code. And I think it can be replace by `public static boolean alwaysFalse() { return false; }`